src/Microsoft.Identity.Web/DownstreamApiSupport/CalledApiOptions.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net.Http;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Options passed-in to call web APIs. To call Microsoft Graph, see rather
+    /// <see cref="MicrosoftGraphOptions"/>.
+    /// </summary>
+    public class CalledApiOptions
+    {
```

Comment:
not a fan of the name on this one. i will need to think about it to get some ideas.

Reply:
Done